runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/renderers/GatewayFormRenderer.tsx

```diff
@@ -145,16 +149,33 @@ const GatewayFormRenderer = () => {
                                 onCancel={() => handleCancel(outgoing)}
                                 onFocus={handleOnFocus}
                             />
-                            <TextField
-                                fullWidth
+                            <AutocompleteWidget
+                                id={'autocomplete-text-field-' + outgoing.id}
+                                handleEvent={true}
+                                onChange={handleExpressionChange}
+                                autocompleteOptions={options}
+                                handleOnBlur={handleOnBlur}
+                                handleOnFocus={handleOnFocus}
                                 label={`${translate(
                                     'Process.Details.Modeler.ActionPanel.Form.Connection.Expression.Expression'
                                 )}`}
-                                name={outgoing.id}
                                 value={outgoing.businessObject.conditionExpression?.body}
-                                onChange={handleExpressionChange}
-                                onBlur={handleOnBlur}
-                                onFocus={handleOnFocus}
+                                name={outgoing.id}
+                                disabled={false}
+                                required={false}
+                                autofocus={false}
+                                readonly={false}
+                                multiple={false}
+                                // WidgetProps required to be passed to the AutocompleteWidget
+                                schema={{}}
+                                uiSchema={{}}
+                                rawErrors={null}
+                                placeholder={''}
+                                options={{}}
+                                formContext={{}}
+                                registry={null}
+                                onBlur={null}
+                                onFocus={null}
```

Comment:
I have a better idea
<img width="930" alt="image" src="https://github.com/runbotics/runbotics/assets/44121335/3495c02b-e6a5-4493-8f3e-7bbb36db9bb4">

then just pass only those props that you really need